examples/gno.land/r/sys/params/unlock.gno

```diff
@@ -0,0 +1,42 @@
+package params
+
+import (
+	"std"
+
+	"gno.land/p/demo/dao"
+	"gno.land/r/gov/dao/bridge"
+)
+
+const (
+	lockSendKey     = "bank_lockSend.string"
+	UnlockSendTitle = "Proposal to unlock the sending functionality for ugnot."
+	LockSendTitle   = "Proposal to lock the sending functionality for ugnot."
+)
+
+func ProposeUnlockSend() uint64 {
+	callback := func() error {
+		std.SetParamString(lockSendKey, "ugnot")
+		return nil
+	}
+	return propose(callback, UnlockSendTitle, "")
+}
+
+func ProposeLockSend() uint64 {
+	callback := func() error {
+		std.SetParamString(lockSendKey, "ugnot")
+		return nil
+	}
+	return propose(callback, LockSendTitle, "")
+}
+
+func propose(callback func() error, title, desc string) uint64 {
+	// The callback function is executed only after the proposal is voted on
+	// and approved by the GovDAO.
+	exe := bridge.GovDAO().NewGovDAOExecutor(callback)
+	prop := dao.ProposalRequest{
+		Title:       title,
+		Description: desc,
+		Executor:    exe,
+	}
+	return bridge.GovDAO().Propose(prop)
+}
```

Comment:
You should check `params.gno`. There are existing generic helpers that make the implementation more concise and less error-prone.

```suggestion
func ProposeLockSend() uint64 {
	exe := NewStringPropExecutor(lockSendKey, "ugnot")
	return propose(exe, LockSendTitle, "")
}
```

The `propose` helper can be added in `params.gno`, not in `unlock.gno`.

In general, I recommend completely deleting the `unlock.gno` file and using the generic helpers to create new proposals. However, if you want to keep them, please try to use the existing helpers and centralize generic elements in the main file.

Reply:
Agree! Done. I will keep the parameter proposal in a separate file. This way, it can be easily isolated and reviewed on GnoWeb.